src/cpp/r/session/RStdCallbacks.cpp

```diff
@@ -613,6 +615,7 @@ void Raddhistory(SEXP call, SEXP op, SEXP args, SEXP env)
 // NOTE: Win32 doesn't receive this callback
 void RSuicide(const char* s)
 {
+   std::cerr << s << std::endl;
```

Comment:
Could you add a comment here indicating why we're emitting this to stderr *and* logging it?